hi-ml-multimodal/src/health_multimodal/image/model/model.py

```diff
@@ -60,22 +59,18 @@ def get_biovil_resnet(pretrained: bool = True) -> ImageModel:
     return image_model
 
 
-@enum.unique
-class ResnetType(str, enum.Enum):
-    RESNET18 = "resnet18"
-    RESNET50 = "resnet50"
+class BaseImageModel(nn.Module, ABC):
```

Comment:
If we're not planning to create any other classes for image models, do we need this?

Reply:
Created a new model type as we discussed.

Reply:
I think this is still unnecessary.